applications/mne_anonymize/mainwindow.h

```diff
@@ -0,0 +1,17 @@
+#ifndef MAINWINDOW_H
```

Comment:
please make this follow our class template

Reply:
I ve deleted the whole class in order to create it from scratch. I'll make sure the new class will use the template.